app/src/app/pages/iconography/iconography.component.html

```diff
@@ -17,5 +17,26 @@
     </div>
   </div>
 
-  <app-slider [items]="sliderItems"></app-slider>
+  <ul class="nav nav-tabs">
+    <li class="nav-item">
+      <a class="nav-link" [ngClass]="{ active: showCurrentIconographyArtworks }" data-toggle="tab" href="#current_iconography_artworks_tab">
+        <i class="mr-1 fa fa-palette"></i>
+        <ng-container>{{notation}}</ng-container>
+      </a>
+    </li>
+    <li class="nav-item"> 
+      <a class="nav-link"[ngClass]="{ active: !showCurrentIconographyArtworks }" data-toggle="tab" href="#child_iconography_artworks_tab">
+        <i class="mr-1 fa fa-palette"></i>
+        <ng-container i18n="Children description@@iconclass_children">Children</ng-container>
```

Comment:
We should probably find a better heading for this, but i have no idea. Furthermore it would be nice to see how many artworks are found for the tab. The latter also applies for the original tab above.